runtimes-common/modules/proc_providing_direct_documentation_feedback.adoc

```diff
@@ -0,0 +1,25 @@
+//[preface]
+[id="providing-direct-documentation-feedback_{context}"]
+= Providing feedback on Red Hat documentation
+
+[role="_abstract"]
+We appreciate your feedback on our technical content and encourage you to tell us what you think.
+If you'd like to add comments, provide insights, correct a typo, or even ask a question, you can do so directly in the documentation.
+
+[NOTE]
+====
+You must have a Red Hat account and be logged in to the customer portal.
+====
+
+To submit documentation feedback from the customer portal, do the following:
+
+. Select the *Multi-page HTML* format.
+. Click the *Feedback* button at the top-right of the document.
+. Highlight the section of text where you want to provide feedback.
+. Click the *Add Feedback* dialog next to your highlighted text.
+. Enter your feedback in the text box on the right of the page and then click *Submit*.
+
+We automatically create a tracking issue each time you submit feedback.
+Open the link that displays after you click *Submit* and start watching the issue or add more comments.
```

Comment:
@inoxx03 please, don't worry about the native English thing. there are plenty of people born in English speaking countries that can't use the language properly (including me at times).  

I still favour _displays_ here. It's something a tech editor once ingrained in me. When describing the action of a dialog or some other kind of GUI, use _displays_ to mean what users actually see on the screen. 

The reasoning is that _appears_ can be more ambiguous. There is also a translation issue for Japanese iirc. It's like _appears_ has more of an indefinite meaning in some other languages.

Reply:
OK, I haven't considered the aspect of translatability here. Thank you for the clarification:) Maybe I should adopt that usage, too. Sounds like sound advice:)

Reply:
I think the correct use of "display" here would be: "Open the link that is displayed after..."
I get confused by transitive vs. intransitive, but if I recall correctly, if you use "display" it needs an object, for example, "The window displays the new certificate" versus "The link is displayed." 

Also, as a side note, the IBM Style Guide says not to use "appear" unless there is no better alternative, since "appear" implies magic or illusion. (I haven't read the latest version of the ISG yet so I hope that advice still applies.) IIRC, I think it presents a translation issue as well.